### PR TITLE
Show clickable reservation list on dashboard and profile

### PR DESCRIPTION
**Pull request by jwalgran**
This pull adds a list of reservations to an individual mapper's profile and dashboard. Clicking a reservation links to the Reservations page with the blockface ID as the URL hash, allowing the map code to fetch the blockface extent from a new endpoint and zoom the map to fit.

I humanized the dates using ``naturalday`` to get expiration dates to display as  'today' and 'tomorrow' when appropriate.

Hiding a portion of the reservation list, then showing it on click, works just like the group list. I refactored the event handling into a common module that uses data-* attributes to drive the behavior.

### CODE REVIEW

**Comment by RickMohr** [comment]
Consider saving a round trip by using the mechanism from 4e1dee2 -- In the reservation page template, put the blockface's extent on the map element as a data attribute and `map.create` will zoom to it for you.

**Comment by kdeloach** [comment]
Looking at this now.

File: src/nyc_trees/apps/users/templates/users/partials/profile/groups.html [hunk 1]
```diff
@@ -6,14 +6,14 @@
...
+   data-class="group-row">
    <div class="row"><div class="col-xs-12"><h5>{{ follow.group.name }}</h5></div></div>
 </a>
 {% endfor %}
 
 {% if follows.hidden_count > 0 %}
-<a data-class="user-profile-groups-show-all" href="javascript:;" class="group block item">
-   <div class="row"><div class="col-xs-12"><h5>+{{ follows.hidden_count }} more group{{ follows.hidden_count|pluralize }}</h5></div></div>
+<a data-class="show-all-list-rows" data-row-data-class="group-row" href="javascript:;" class="group block item">
+   <div class="row"><div class="col-xs-12"><h5>+{{ follows.hidden_count }} more group{{ follows.hiduden_count|pluralize }}</h5></div></div>
```
Comment (kdeloach) [inline src/nyc_trees/apps/users/templates/users/partials/profile/groups.html]: Typo (hiduden_count)?

File: src/nyc_trees/js/src/bindShowAllRowsHandlers.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+"use strict";
```
Comment (kdeloach) [inline src/nyc_trees/js/src/bindShowAllRowsHandlers.js]: Consider moving to `js/src/lib`.

**Comment by kdeloach** [comment]
I am logged in as a user with blockface reservations but I don't see any reservations listed on my dashboard or profile page. How should I test this?

**Comment by jwalgran** [comment]
Does your user have the ``invividual_mapper`` flag set?

**Comment by kdeloach** [comment]
Nope! I'll set that flag and try again. Thanks.

**Comment by kdeloach** [comment]
Worked perfectly after setting `invividual_mapper`.

**Comment by kdeloach** [comment]
+1

**Pull request merged**